### PR TITLE
fix: Configuring storage classes

### PR DESCRIPTION
**Pull request by tolusha**
## What does this pull request change?
Fixes article related to configuring storage classes

## What issues does this pull request fix or reference?
https://issues.redhat.com/browse/RHDEVDOCS-5516

## Specify the version of the product this pull request applies to
main

## Pull Request checklist

The author and the reviewers validate the content of this pull request with the following checklist, in addition to the [automated tests](code_review_checklist.adoc).

- Any procedure:
  - [x] Successfully tested.
- Any page or link rename:
  - [ ] The page contains a redirection for the previous URL.
  - Propagate the URL change in:
    - [ ] Dashboard [default branding data](https://github.com/eclipse-che/che-dashboard/blob/main/packages/dashboard-frontend/src/services/bootstrap/branding.constant.ts)
    - [ ] Chectl [constants.ts](https://github.com/che-incubator/chectl/blob/main/src/constants.ts)
- [ ] Builds on [Eclipse Che hosted by Red Hat](https://workspaces.openshift.com).
- [ ] the *`Validate language on files added or modified`* step reports no vale warnings.

### CODE REVIEW

**Comment by tolusha** [comment]
@ibuziuk 
Please have a look. I've removed some sentences that seem not relevant anymore.

File: modules/administration-guide/pages/installing-che-using-storage-classes.adoc [hunk 1]
```diff
@@ -46,43 +36,17 @@ Use CheCluster Custom Resource definition to define storage classes:
...
-  accessModes:
-    - ReadWriteOnce
-  hostPath:
-    path: "/data/che/workspace"
-----
-
-. Bind the persistent volume:
-+
-[subs="+quotes,+attributes"]
+      perUserStrategyPvcConfig:
```
Comment (ibuziuk) [inline modules/administration-guide/pages/installing-che-using-storage-classes.adoc:39]: @tolusha can we add more yaml here? `spec:storage:` I mean to make it clear what part of CR is expected to be updated

**Review by ibuziuk — APPROVED** [review]
LGTM one minor suggestion to make the yaml sample more explicit

File: modules/administration-guide/pages/installing-che-using-storage-classes.adoc (hunk 1 above)
Comment (tolusha) [reply modules/administration-guide/pages/installing-che-using-storage-classes.adoc:39]: Diff confuses, but the whole CR is there:

screenshot-64b12ece30412d0336828eb1--eclipse-che-docs-pr netlify app-2023 07 17-14_53_19

File: modules/administration-guide/pages/installing-che-using-storage-classes.adoc [hunk 2]
```diff
@@ -31,9 +22,8 @@ To configure {prod-short} to use a configured infrastructure storage, install {p
 
 * {prod-short} does not create persistent volumes in the infrastructure.
 * {prod-short} uses persistent volume claims (PVC) to mount persistent volumes.
-* The {prod-short} server creates  persistent volume claims.
 +
-A user defines a storage class name in the {prod-short} configuration to use the storage classes feature in the {prod-short} PVC. With storage classes, a user configures infrastructure storage in a flexible way with additional storage parameters. It is also possible to bind a static provisioned persistent volumes to the {prod-short} PVC using the class name.
+A user defines a storage class name in the {prod-short} configuration to use the storage classes feature in the {prod-short} PVC.
```
Comment (deerskindoll) [inline modules/administration-guide/pages/installing-che-using-storage-classes.adoc]:
```suggestion
Define a storage class name in the {prod-short} configuration to use the storage classes feature in the {prod-short} PVC.
```

File: modules/administration-guide/pages/installing-che-using-storage-classes.adoc [hunk 3]
```diff
@@ -7,16 +7,7 @@
...
-
-* A special host path
-* A storage capacity
-* A volume mod
-* Mount options
-* A file system
-* An access mode
-* A storage type
-* And many others
+To configure {prod-short} to use a configured infrastructure storage, install {prod-short} using storage classes. This is especially useful when a user wants to bind a persistent volume provided by a non-default provisioner.
```
Comment (deerskindoll) [inline modules/administration-guide/pages/installing-che-using-storage-classes.adoc]:
```suggestion
To configure {prod-short} to use a configured infrastructure storage, install {prod-short} using storage classes. This is especially useful when you want to bind a persistent volume provided by a non-default provisioner.
```

**Review by deerskindoll — COMMENTED** [review]
two small suggestions

**Review by deerskindoll — APPROVED** [review]
lgtm

**Pull request merged**